README.md

```diff
@@ -94,11 +129,18 @@ If you're interested in contributing to the project, please see our [Contributin
 
 ## Other Options
 
-TODO
+### Docker Support
 
-See [Makefile](./Makefile)
+Docker support is available to provide a consistent build environment across different operating systems. This is particularly useful for:
 
+- **macOS users**: Avoids issues with hardcoded compiler names in dependencies like `lwext4_rust`
+- **Windows users**: Provides a Linux build environment without WSL
+- **CI/CD**: Ensures consistent builds across different environments
+
+See the [Dockerfile](./Dockerfile) and [docker-compose.yml](./docker-compose.yml) for details.
```

Comment:
This "Other Options" part is reserved for `make` options, not build environment.